web/src/main/java/com/epam/rft/atsy/web/controllers/FileDownloaderController.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.rft.atsy.web.controllers;
+
+import com.epam.rft.atsy.service.CandidateService;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.util.FileCopyUtils;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URLConnection;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletResponse;
+
+@Controller
+public class FileDownloaderController {
+  private static final String CONTENT_DISPOSITION = "Content-Disposition";
+  private static final String MIME_TYPE = "application/octet-stream";
+  private static final String INLINE_FORMAT = "inline; filename=\"";
+
+  @Resource
+  private CandidateService candidateService;
+
+  @RequestMapping(value = "/secure/candidate/fileDownload/{candidateId}", method = RequestMethod.GET)
+  public void downloadFile(@PathVariable("candidateId") Long candidateId,
+                           HttpServletResponse response) throws IOException {
+
+    String path = candidateService.getCVPathByCandidateId(candidateId);
```

Comment:
What if the candidate does not have a file uploaded? 

This should be checked and handled gracefully.